src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/common.targets

```diff
@@ -65,6 +66,12 @@
         <RelativePath>$([System.IO.Path]::GetFileName('$(_MvcRazorOutputFullPath)'))</RelativePath>
       </ResolvedFileToPublish>
     </ItemGroup>
+
+    <ItemGroup Condition="'$(MvcRazorExcludeRefAssembliesFromPublish)'=='true'">
+      <ResolvedFileToPublish
+        Remove="%(ResolvedFileToPublish.Identity)"
+        Condition="'%(ResolvedFileToPublish.RelativePath)'=='$(RefAssembliesFolderName)\%(Filename)%(Extension)'" />
```

Comment:
Have you tested this on xplat to ensure .RelativePath is a backslash, not forward slash?